sysutils/dosfsutils/Portfile

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        dosfstools dosfstools 4.1 v
+github.tarball_from releases
+
+name                dosfsutils
```

Comment:
Why rename the pot from what upstream calls it, dosfstools ?
I would favour sticking with the upstream name, and thus just remove this line.
You even refer to it as dosfstools in the description....